packages/Tag/stories/Tag.stories.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable react/no-array-index-key */
+import React from "react";
+import CalendarIcon from "@paprika/icon/lib/Calendar";
+import ExclamationCircle from "@paprika/icon/lib/ExclamationCircle";
+import tokens from "@paprika/tokens";
+import Avatar from "../../Avatar/src";
+
+import Tag, { Tags } from "../src";
+import * as types from "../src/types";
+
+export default {
+  title: "Tag",
+  component: Tag,
+};
+
+export function TagsExample() {
+  const handleRemove = () => {
+    console.log("remove event");
```

Comment:
These are nice to have, but can we use the Storybook actions panel instead of console.log?